### ISSUE TITLE
[18.0] Resubscribe auditolog rules

### ISSUE THREAD

**Issue by tarteo** [onesteinbv/addons-container#348]
I've removed the subscription as it results in errors, the search domain here should be more restrictive to the rules only introduced by the module itself the same for hr_accessibility

https://github.com/onesteinbv/addons-curq/commit/b8654a9063f39cb5b0a3e56e1521c3f227453337

**Comment by tarteo** [comment]
https://github.com/onesteinbv/addons-curq/commit/26a9575cfe3fc5547175aeaf9b676dc2e9a9806c
